p2p/src/authenticated/lookup/mod.rs

```diff
@@ -215,11 +232,8 @@ mod tests {
             oracles.push(oracle);
 
             // Register basic application
-            let (mut sender, mut receiver) = network.register(
-                0,
-                Quota::per_second(NZU32!(5)), // Ensure we hit the rate limit
```

Comment:
We were willing to send `All` every 100ms (which would've led to rate limiting).

Reply:
(I don't suspect this is related, but another worthwhile cleanup)